collector/fixtures/e2e-output.txt

```diff
@@ -2882,6 +2883,36 @@ node_softnet_times_squeezed_total{cpu="0"} 1
 node_softnet_times_squeezed_total{cpu="1"} 10
 node_softnet_times_squeezed_total{cpu="2"} 85
 node_softnet_times_squeezed_total{cpu="3"} 50
+# HELP node_tape_io_now The number of I/Os currently outstanding to this device.
+# TYPE node_tape_io_now gauge
+node_tape_io_now{device="st0"} 1
+# HELP node_tape_io_others_total The number of I/Os issued to the tape drive other than read or write commands. The time taken to complete these commands uses the following calculation io_time_seconds_total-read_time_seconds_total-write_time_seconds_total
+# TYPE node_tape_io_others_total counter
+node_tape_io_others_total{device="st0"} 1409
+# HELP node_tape_io_time_seconds_total The amount of time spent waiting (in nanoseconds) for all I/O to complete (including read and write). This includes tape movement commands such as seeking between file or set marks and implicit tape movement such as when rewind on close tape devices are used.
```

Comment:
The `(in nanoseconds)` part is not correct right? Sure, the underlying data source uses that but that's irrelevant for the consumer of the metric.

Reply:
It is the kernel doc description. It would tell the user the metric resolution (nanoseconds converted to seconds), but I can remove it.